backend/npm-debug.log

```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```

Comment:
I think this file shouldn't be pushed to the repo.

Reply:
This is handled.